.github/workflows/deploy_staging.yml

```diff
@@ -66,7 +68,7 @@ jobs:
           path: ./public
       - name: Set DESTINATION_FOLDER for branch
         run: |
-          echo "DESTINATION_FOLDER=$(echo $GITHUB_SHA | cut -c 1-6)" >> $GITHUB_ENV
+          echo "DESTINATION_FOLDER=$(echo $PR_NUMBER-$GITHUB_SHA | cut -c 1-11)" >> $GITHUB_ENV
```

Comment:
I'm curious about the change to use PR number as well as the Git SHA. Were we running into issues with just the SHA?

Reply:
Since a given PR could have multiple builds (and therefore multiple SHAs and therefore multiple artifacts), this would allow us to group all the artifacts by PR and remove them all in a single command. :)

Reply:
oh nice! cool